modules/microshift-data-cleaning-ovn-data.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-cleanup-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-data-cleaning-ovn-data_{context}"]
+= Cleaning the OVN data
+
+Cleanup the OVN data by running the following script. This script is useful to reset OVN settings after modifying network configurations. This script performs the following cleanup actions:
+
+* Stop and disable {microshift-short} services
+* Delete all {microshift-short} pods
+* Delete OVN configuration
+
+.Procedure
+
+. Execute the `microshift-cleanup-data` script, by running the following command.
```

Comment:
```suggestion
. To clean up the OVN-K data by running the `microshift-cleanup-data` script with the `--ovn` argument, enter the following command:
```
Alternative syntax that meets the same doc style requirement. Another thought. I'd pick one form and stick to it throughout.